chat-plugins/info.js

```diff
@@ -1821,10 +1822,10 @@ var commands = exports.commands = {
 			}
 			return this.sendReplyBox("Random number " + num + "x(1 - " + faces + "): " + rolls.join(", ") + "<br />Total: " + total);
 		}
-		if (target && isNaN(target) || target.length > 21) return this.sendReply("The max roll must be a number under 21 digits.");
-		var maxRoll = (target) ? target : 6;
-		var rand = Math.floor(maxRoll * Math.random()) + 1;
-		return this.sendReplyBox("Random number (1 - " + maxRoll + "): " + rand);
+		if (isNaN(target) || target >= 1e22) return this.sendReply("The max roll must be a number under 21 digits.");
```

Comment:
I'd say
<code> if (!Number.isSafeInteger(target) || target <=) return this.sendReply(""The max roll must be a natural number up to " + Number.MAX_SAFE_INTEGER + ".");